charts/gitprime-app/v2.0.0/questions.yml

```diff
@@ -0,0 +1,451 @@
+        default: "/mnt/repo-storage"
+      - variable: dataPipeline.volumes.podMountDirectory
+        type: string
+        label: "Directory on Pod"
+        description: "The directory inside the pod to mount the volume."
+        required: true
+        default: "/var/repos"
+
+  # New Repo Worker Settings
+  - variable: dataPipeline.newRepoWorker.replicaCount
+    type: string
+    required: true
+    group: "Data Pipeline / New Repo Workers"
+    label: Number of Pods for New Repo Workers
+    description: The number of pods to deploy for the workers that handle new repositories.
+    default: 1
+
+  - variable: dataPipeline.newRepoWorker.javaOptions
+    type: string
+    required: true
+    group: "Data Pipeline / New Repo Workers"
+    label: New Repo Worker Java Options
+    description: The Java options for the new repository workers.
+    default: "-d64 -Xms2g -Xmx4g"
+
+  - variable: dataPipeline.newRepoWorker.processorThreadCount
+    type: string
+    required: true
+    group: "Data Pipeline / New Repo Workers"
+    label: New Repo Worker Thread Count
+    description: The number of threads to use for the new repository workers.
+    default: 2
+
+  - variable: dataPipeline.newRepoWorker.maxConcurrentJobs
+    type: string
+    required: true
+    group: "Data Pipeline / New Repo Workers"
+    label: New Repo Worker Concurrent Jobs
+    description: The number of concurrent jobs for the new repository workers.
+    default: 1
+
+  - variable: dataPipeline.newRepoWorker.maxCommitCount
+    type: string
+    required: true
+    group: "Data Pipeline / New Repo Workers"
+    label: New Repo Maximum Commit Count
+    description: The maximum number of commits each worker will handle before resting.
+    default: 600
+
+  - variable: dataPipeline.newRepoWorker.resources.requests.cpu
+    type: string
+    required: true
+    group: "Data Pipeline / New Repo Workers"
+    label: New Repo Worker Requested CPU
+    description: The number of CPUs the new repository workers will request.
+    default: "2"
+
+  - variable: dataPipeline.newRepoWorker.resources.requests.memory
+    type: string
+    required: true
+    group: "Data Pipeline / New Repo Workers"
+    label: New Repo Worker Requested Memory
+    description: The maximum amount of memory a new repository worker will request.
+    default: "4Gi"
+
+  - variable: dataPipeline.newRepoWorker.resources.limits.cpu
+    type: string
+    required: true
+    group: "Data Pipeline / New Repo Workers"
+    label: New Repo Worker CPU Limit
+    description: The maximum number of CPUs the new repository workers can use.
+    default: "4"
+
+  - variable: dataPipeline.newRepoWorker.resources.limits.memory
+    type: string
+    required: true
+    group: "Data Pipeline / New Repo Workers"
+    label: New Repo Worker Memory Limit
+    description: The maximum amount of memory a new repository worker can use.
+    default: "8Gi"
+
+  - variable: dataPipeline.newRepoWorker.requeueAOD
```

Comment:
RequeueAOD is no longer needed.  I believe this next version of the app "embeds" this decision.  That is due to, in part, a separation of the commit processing "message consumer" and a new AOD "message consumer".